### DISCUSSION TITLE
is there custom configurations, I can pass some header to traces,but do not require code changes

### DISCUSSION THREAD

**Discussion by liyuntian** [apache/skywalking#10633] [General]
skywalking version:9.2.0
I can get some solutions from this blog  https://www.cnblogs.com/duanxz/p/15607843.html.but is there some configuration or parameter we can use directly without any modify.
In OTEL we have the same function
OTEL_INSTRUMENTATION_METHODS_INCLUDE="com.xxxx.yun.xxx.application.xxx.xxx[toCreate,toModify]"
 OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_CLIENT_RESPONSE="x-ca-xxx-id"

**Comment by wu-sheng** [comment]
I think this is you are looking for https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/application-toolkit-trace-correlation-context/

**Comment by wu-sheng** [comment]
We don't recommend things like you mentioned like a config. The value is mutant, there is no point to make things complex.

**Reply by wu-sheng** [reply]
If you could write things like a conig, why can't you write codes(manual SDK) or a plugin.

**Reply by liyuntian** [reply]
I can  codes or a plugin to pass some header to traces.but If I can use some configuration without any code, I can modify client services at any time.now I can node in my client service.
![Uploading 1680515091942.jpg…]()

**Reply by wu-sheng** [reply]
Again and again. I had repeated myself a lot of times. Don't use another project to require SkyWalking. That is pointless.
Comparing and suggesting to do something in SkyWalking mostly mean you are going to implement this and submit a pull request to make it works as the background.
Otherwise, I can't see why we need to discuss this.

**Reply by wu-sheng** [reply]
Meanwhile, you attached a thing we can't see.
